streampark-console/streampark-console-webapp/src/views/flink/app/components/AppView/StartApplicationModal.vue

```diff
@@ -89,6 +90,26 @@
         ifShow: ({ values }) => values.startSavePointed,
         required: true,
       },
+      {
+        field: 'restoreMode',
+        label: 'restore mode',
+        component: 'Select',
+        defaultValue: RestoreModeEnum.NO_CLAIM,
+        componentProps: {
+          options: [
+            { label: 'CLAIM', value: RestoreModeEnum.CLAIM },
+            { label: 'NO_CLAIM', value: RestoreModeEnum.NO_CLAIM },
+            { label: 'LEGACY', value: RestoreModeEnum.LEGACY },
+          ],
+        },
+        afterItem: () =>
```

Comment:
Have we checked the version number of Flink here?

Reply:
done